translib/sys_app.go

```diff
@@ -43,6 +43,13 @@ type SysApp struct {
 	procTable   map[uint64]dbEntry
 }
 
+type reqType int
```

Comment:
Do we really need this change in this file? It's not been used in xfmr_intf.go

Reply:
Used to be used, seems to be removed with one of the code changes to remove unnecessary code. Will double check & remove.

Reply:
@rathnasabapathyv It is required in the sys_app.go file and pfm_app.go file. It was earlier defined in the intf_app.go file and since we deleted that file, we were seeing the following errors so had to redefine it here:

```
./pfm_app.go:38:28: undefined: dbEntry
./pfm_app.go:168:39: undefined: dbEntry
./pfm_app.go:184:39: undefined: dbEntry
./sys_app.go:42:25: undefined: dbEntry
./sys_app.go:43:25: undefined: dbEntry
./sys_app.go:269:36: undefined: dbEntry
./sys_app.go:285:33: undefined: dbEntry
./sys_app.go:290:34: undefined: dbEntry
./sys_app.go:307:24: undefined: dbEntry
make[3]: *** [Makefile:59: /sonic/src/sonic-mgmt-common/build/tests/translib/testapp.test] Error 2
```

Reply:
As part of this enhancement, we don't have any enhancements in `pfm_app.go` & `sys_app.go`. Without this change, existing community code `pfm_app.go`, how it's compiling?

Reply:
Looks like in existing community code, these two typedefs `reqType` & `dbEntry` were present earlier in `intf_app.go`, which has been removed by us as part of this PR and we have to park it somewhere. It's ok then.